docs/integrations/mirroring.md

```diff
@@ -24,34 +24,34 @@ A Mirroring Integration's YAML file should have the following parameters (under
+- `get-remote-data` - this command gets new information about the incidents in the remote system and updates *existing* incidents in Cortex XSOAR. If an API rate limit error occures, this method should return the exact error - `"API rate limit"`, so that the sync loop will start from the failed incident.
+This command is executed every 1 minute for each individual **incident** fetched by the integration. 
 - `update-remote-system` - this command updates the remote system with the information we have in the mirrored incidents within Cortex XSOAR. This command is executed whenever the individual incident is changed in Cortex XSOAR.
 - `get-mapping-fields` - this command pulls the remote schema for the different incident types, and their associated incident fields, from the remote system. This enables users to map XSOAR fields to the 3rd-party integration fields in the outgoing mapper. This command is being called when selecting the **Select schema** option in the **Get data** configuration in a classifier or a mapper.
 
 ## How to implement mirroring functions
 You can implement the following functions, using the classes described below, which are globally available through the CommonServerPython file.
 
 ### get-remote-data
-* GetRemoteDataArgs - this is an object created to maintain all the arguments you receive from the server in order to use this command.
+* **GetRemoteDataArgs** - this is an object created to maintain all the arguments you receive from the server in order to use this command.
 Arguments explanation:
-  - remote_incident_id - contains the value of the dbotMirrorId field, which represents the id of the incident in the external system.
-  - last_update - the time the incident was last updated - our recommendation would be to use the arg_to_timestamp function to parse it into a timestamp.
-* GetRemoteDataResponse - this is the object that maintains the format in which you should order the results from this function. You should use return_results on this object to make it work.
+  - *remote_incident_id* - contains the value of the dbotMirrorId field, which represents the id of the incident in the external system.
+  - *last_update* - the time the incident was last updated - our recommendation would be to use the arg_to_timestamp function to parse it into a timestamp.
+* **GetRemoteDataResponse** - this is the object that maintains the format in which you should order the results from this function. You should use return_results on this object to make it work.
 Arguments explanation:
-  - mirrored_object - this is essentially the object(dict) of whatever you are trying to mirror - incident in most cases.
-  - entries - a list of entries to add to your incident.
+  - *mirrored_object* - this is essentially the object(dict) of whatever you are trying to mirror - incident in most cases.
+  - *entries* - a list of entries to add to your incident.
```

Comment:
We should expand on this. For example, if there are only entries (no change to incident), you can return `{}` as the first entry. If you want to close the incident, you can return an entry with `{"Contents": {"dbotIncidentClose": True, "closeReason": "some reason", "closeNotes": "Some note"}`. If you want to re-open a closed incident, you should return `{"Contents": {"dbotIncidentReopen": True}}. Also, worth noting that we support the full entry syntax including marking it as note.